shared/global-errors/index.native.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+import React, {Component} from 'react'
+import {Box, Text, Icon, HOCTimers, NativeScrollView, NativeTouchableWithoutFeedback} from '../common-adapters/index.native'
+import {globalStyles, globalColors, globalMargins} from '../styles'
+
+import type {Props} from './index'
+
+type Size = 'Closed' | 'Small' | 'Big'
+type State = {
+  size: Size,
+  cachedSummary: ?string,
+  cachedDetails: ?string,
+}
+
+class GlobalError extends Component<void, Props, State> {
```

Comment:
I'm talking more about the logic of hiding after a timeout and using state to keep the message, not the views. If you think that the mobile stuff will deviate from desktop thats fine too.